test/utils.js

```diff
@@ -0,0 +1,58 @@
+const Matroid = require('../lib/matroid');
```

Comment:
Update this to 

```javascript
require('../dist/matroid');
```
to make sure we're testing the code the user will be using

Reply:
Fixed!